index.js

```diff
@@ -7,24 +7,39 @@ const fs = require('fs');
 
 async function run() {
   try {
-    const msys2base = await tc.downloadTool('http://repo.msys2.org/distrib/x86_64/msys2-base-x86_64-20190524.tar.xz');
-    const tar = await io.which('tar', true);
+    if (process.platform != 'win32') {
```

Comment:
I think this should be `!==`

Reply:
Thanks! I'm not a JS developer, so might do many mistakes like this...